packages/trezor-user-env-link/src/websocket-client.ts

```
@@ -97,7 +97,7 @@ export class WebsocketClient extends WebsocketClientBase<WebsocketClientEvents>
                 this.emit('firmwares', resp.firmwares);
             } else {
                 if (!resp.success) {
-                    throw new Error(`websocket_error_message: ${resp.error.message || resp.error}`);
+                    console.log(`websocket_error_message: ${resp.error.message || resp.error}`);
```

Are you sure about this? I am not sure it's a good idea to ignore the error like this...

---

lets solve this in separartion, commit dropped and moved here https://github.com/trezor/trezor-suite/pull/17277